samples/word/99-preview-apis/manage-footnotes.yaml

```diff
@@ -0,0 +1,264 @@
+id: word-manage-footnotes
+name: Manage footnotes
+description: Basic operations using footnotes.
```

Comment:
Same name and description suggestions are the previous one: name it "Footnotes" and elaborate a bit more in the description.